test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -451,5 +477,92 @@ private static void DeleteDirectory(string path)
 
             }
         }
+
+        //temporary added methods for processing deps.json files for patching
+        private void TempPatching(string patchDir)
+        {
+            var prefix = patchDir;
+            string[] filenames = new string[3] { "dotnet.deps.json", "MSBuild.deps.json", "NuGet.CommandLine.XPlat.deps.json" };
+            string[] fullnames = new string[3];
+            for(int i = 0; i < filenames.Length; i++) 
+            {
+                fullnames[i] = prefix + Path.DirectorySeparatorChar + filenames[i];
+            }
+            TempPatchingDepsJsonForNewlyAddedDlls(fullnames);
+
+        }
+        private void TempCopyNewlyAddedDlls(string[] dlls, string pathCopyFrom, string PathCopyTo)
+        {
+            foreach (var dll in dlls) 
+            {
+                var copyFromFullName = Path.Combine(pathCopyFrom, dll);
+                var copyToFullName = Path.Combine(PathCopyTo, dll);
+                File.Copy(copyFromFullName, copyToFullName);
+            }
+        }
+        private void TempPatchingDepsJsonForNewlyAddedDlls(string[] filePaths)
+        {
+            string nugetBuildTasksName = "NuGet.Build.Tasks/5.3.0-rtm.6251";
+            foreach (string file in filePaths)
+            {
+                JObject JsonFile = GetJson(file);
+
+                JObject targets = null;
+                targets = JsonFile.GetJObjectProperty<JObject>("targets");
+
+                JObject netcoreapp50 = null;
+                netcoreapp50 = targets.GetJObjectProperty<JObject>(".NETCoreApp,Version=v5.0");
+
+                JObject NuGet_Build_Tasks = null;
+                NuGet_Build_Tasks = netcoreapp50.GetJObjectProperty<JObject>(nugetBuildTasksName);
+
+                JObject runtime = null;
+                runtime = NuGet_Build_Tasks.GetJObjectProperty<JObject>("runtime");
+
+                var jproperty = new JProperty("lib/netstandard2.1/System.Security.Cryptography.Pkcs.dll",
+                            new JObject
+                                {
+                                    new JProperty("assemblyVersion","4.0.4.0"),
+                                    new JProperty("fileVersion", "5.0.19.47301"),
```

Comment:
Would it be a more robust solution to read these values from System.Security.Cryptography.Pkcs.dll instead of hard-coding them here?  I don't know how long this temporary solution will be around...

Reply:
How do you read it from the assembly?

Reply:
nevermind, my google-fu finally succeeded :)

Reply:
I believe this is resolved. Do you mind taking another look before I mark this as resolved?